### PR TITLE
Use esbuild for bundling

### PR DESCRIPTION
**Pull request by Pessimistress**
For #242

### CODE REVIEW

File: scripts/bundle.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+ * e.g. @deck.gl/core is not bundled with @deck.gl/geo-layers
+ */
+function getExternals(packageInfo) {
+  let externals = {
+    // Hard coded externals
+    // 'h3-js': 'h3'
+  };
+  const {peerDependencies = {}} = packageInfo;
+
+  for (const depName in peerDependencies) {
```
Comment (chrisgervang) [inline scripts/bundle.js:22]: We've got the hubble packages in main's dependencies. Should they actually be in peerDependencies?

Comment (Pessimistress) [reply scripts/bundle.js:22]: So if something is in dependencies, then it's bundled with. If something is in dependencies, then it's bundled without. The difference is 

```html
<script src="https://unpkg.com/deck.gl@^8.0.0/dist.min.js"></script>
<script>
  // core, layers, and geo-layers are dependencies of main
  const {MVTLayer} = deck;
</script>
```

vs

```html
<script src="https://unpkg.com/@deck.gl/core@^8.0.0/dist.min.js"></script>
<script src="https://unpkg.com/@deck.gl/layers@^8.0.0/dist.min.js"></script>
<script src="https://unpkg.com/@deck.gl/geo-layers@^8.0.0/dist.min.js"></script>
<script>
  // core and layers are peer dependencies of geo-layers
  const {MVTLayer} = deck;
</script>
```

Comment (chrisgervang) [reply scripts/bundle.js:22]: I'm confused, typo? Which is peer dependencies?

Comment (Pessimistress) [reply scripts/bundle.js:22]: When A is in B's peer dependencies, you need to explicitly include both bundle A and bundle B for B to work.
When A is in B's dependencies, you only need to include bundle B for B to work.

**Pull request closed without merging**